### PR TITLE
Clarify deleting persistent volumes.

### LINKED ISSUE #603: [doc] 5.4  Test Storage Class
5.4  Test Storage Class

https://documentation.suse.com/suse-cap/1.5/single-html/cap-guides/#sec-cap-teststorageclass-prod

When doing this test, and you get to the section where it says to clean up.  "Delete your volume claims when you're finished: "

If you follow the commands, it asks you to delete the persistent volume  (pv) first, and then the persistent volume claim (pvc).  In my testing, this is backwards.  I had to delete the pvc first, and then delete the pv.  When I tried to delete the pv first, the command hangs.  But once I deleted the pvc first, the pv went away easily.

### PR DESCRIPTION
**Pull request by KucharczykL**
Closes #603.

### CODE REVIEW

File: xml/cap_depl_install_caasp.xml [hunk 1]
```diff
@@ -410,12 +410,18 @@ pvc/test-sc-persistent   Bound     pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c   1G
...
 
-<screen>&prompt.user;kubectl delete pv/pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c
-persistentvolume "pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c" deleted
-
-&prompt.user;kubectl delete pvc/test-sc-persistent
+<screen>&prompt.user;kubectl delete pvc/test-sc-persistent
 persistentvolumeclaim "test-sc-persistent" deleted</screen>
 
+  <tip>
+    <title>Deleting persistent volume</title>
```
Comment (btat) [inline xml/cap_depl_install_caasp.xml]: Needs to be updated to title-style capitalization

https://doc.opensuse.org/products/opensuse/Styleguide/opensuse_documentation_styleguide_sd/#sec.capitalization

File: xml/cap_depl_install_caasp.xml [hunk 2]
```diff
@@ -410,12 +410,18 @@ pvc/test-sc-persistent   Bound     pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c   1G
...
+    <para>
+      Usually you have to delete the persistent volume manually but
+      in this case the persistent volume has <literal>reclaimPolicy: Delete</literal>
+      set and thus it is done automatically.
```
Comment (btat) [inline xml/cap_depl_install_caasp.xml]: Might need to reword the start of the sentence (the "Usually you ... manually" portion) slightly. A reclaim policy is either `Retain` or `Delete` (with `Delete` being the default) so the current phrasing at is not entire accurate.

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming

Comment (KucharczykL) [reply xml/cap_depl_install_caasp.xml]: Changed the wording, let me know if it's better this way.

**Comment by btat** [comment]
👍

**Pull request merged**